sbnana/CAFAna/Analysis/Fit.cxx

```diff
@@ -463,6 +471,94 @@ namespace ana
     return ret;
   }
 
+  //----------------------------------------------------------------------
+  TH1* Profile(const IExperiment* expt,
+	       osc::IOscCalcAdjustable* calc, const ISyst* s,
+	       int nbinsx, double minx, double maxx,
+	       double input_minchi,
```

Comment:
Something has gone wrong with the indentation of the argument here, at least in github's view. I suspect there are tabs.